t/harness

```diff
@@ -366,15 +344,21 @@ if (@ARGV) {
         push @$which, 'benchmark' if $ENV{PERL_BENCHMARK};
         push @$which, 'bigmem' if $ENV{PERL_TEST_MEMORY};
 
+        my @extracted = ();
         if (@next) {
-            my $next = { par => '{' . join (',', @next) . '}/*.t' };
-            @next = _extract_tests ($next);
+            foreach my $next (@next) {
+                push @extracted, glob ("$next/*.t");
+            }
+            @next = @extracted;
             push @tests, @next;
             push @seq, _compute_tests_and_ordering(\@next)->@*;
         }
 
-        my $last = { par => '{' . join (',', @last) . '}/*.t' };
-        @last = _extract_tests ($last);
+        @extracted = ();
+        foreach my $last (@last) {
+            push @extracted, glob ("$last/*.t");
+        }
+        @last = @extracted;
```

Comment:
doesnt this end up as: `@last = map { glob("$_/*.t") } @last;` (same above with `@next`?)

Reply:
Yes, thanks.  There seem to be other places where foreach is preferred in this file, so I thought I was going with the flow, but there are a number of other usages of map as well.  So I've pushed a revision that just uses map, which does look simpler overall.